src/test/java/com/znsio/common/JsonPlaceholder.feature

```diff
@@ -0,0 +1,21 @@
+Feature: JsonPlaceholder Utilities
```

Comment:
Add this method to the common.feature No need for any new feature file.

Reply:
Removed above method and added 'query' column in Examples section.